app/src/ui/branches/branches-container.tsx

```diff
@@ -65,17 +67,23 @@ export class BranchesContainer extends React.Component<
     }
   }
 
-  private onFilterKeyDown = (event: React.KeyboardEvent<HTMLInputElement>) => {
+  private closeFoldoutOnEsc = (when: () => boolean) => (
```

Comment:
:lipstick: could this be named to something more descriptive? `shouldCloseFoldout`?

Reply:
Sorry, I actually meant the `when` parameter here, not the function name 🤦‍♂️